### PR TITLE
test: add test cases on the extension of the cri

### PR DESCRIPTION
**Pull request by starnop**
Signed-off-by: Starnop <starnop@163.com>

test: add test cases on the extension of the cri

### CODE REVIEW

File: pkg/framework/util.go [hunk 1]
```diff
@@ -52,6 +52,9 @@ const (
 	// DefaultContainerImage is the default image for container using
 	DefaultContainerImage string = "busybox:1.28"
 
+	// DefaultContainerVolumeImage is the default image for container with volume using
```
Comment (allencloud) [inline pkg/framework/util.go]: Where to use it?

Comment (Starnop) [reply pkg/framework/util.go]: Test whether to support the ability to acquire the volumes of the Image when getting image status.

File: cmd/crictl/volume.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
```
Comment (HusterWan) [inline cmd/crictl/volume.go]: add blank line

File: cmd/crictl/volume.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	pb "github.com/alibaba/pouch/cri/apis/v1alpha2"
```
Comment (HusterWan) [inline cmd/crictl/volume.go]: add blank line

**Pull request merged**